src/components/ContactCard/ContactCard.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { css, cx } from '@emotion/css';
+import { tokens } from '../../theme/tokens';
+import { Typography } from '../Typography';
+import { Link } from '../Link';
+
+type ContactCardProps = {
+  src?: string;
+  title: string;
+  fullName?: string;
+  location?: string;
+  telephone?: string;
+  mail: string;
+  className?: string;
+};
+
+export const ContactCard = React.forwardRef<HTMLDivElement, ContactCardProps>(function ContactCard(
+  { src, title, fullName, location, telephone, mail, className },
```

Comment:
this is quite a lot of props tbh. wdyt on passing object here? something like `cardDetails` and work with object?
usage example
`<ContactCard cardDetails={ContactCardProps}/>`
instead of 
`<ContactCard title={title} mail={mail}...>`

Reply:
I thought about this as I wasn't too convinced about it, but now that you also bring it up I'll clean it 👍

Reply:
You can check it out again if you want :)